### PR TITLE
Fix/platform interaction stability improvements

### PR DESCRIPTION
**Pull request by Sergey-Makarov**
Behavioural changes:
- `MediaCodecInfo` from `CodecInfoProvider` is now guaranteed to have `capabilities` of type `List<String>` instead of `List<String?>`
- `DevicePersonalizationInfoProvider.availableLocales()` is now guaranteed to return `Array<String>` instead of `Array<String?>`
It is achieved by converting nullable strings to `"null"` string. This way fingerprints are guaranteed to not be affected.

### CODE REVIEW

File: fingerprint/src/main/java/com/fingerprintjs/android/fingerprint/tools/logs/Messages.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.fingerprintjs.android.fingerprint.tools.logs
+
+internal fun Logger.ePleaseReport(throwable: Throwable?) {
+    e(
+        msg = "Unexpected error occurred. Feel free to create an issue on Github.",
```
Comment (Alexey-Verkhovsky) [inline fingerprint/src/main/java/com/fingerprintjs/android/fingerprint/tools/logs/Messages.kt:5]: Let's mark this message with "... on Github repository of the fingerprintjs-android library." for the cases when all the stacktrace will be obfuscated.

File: fingerprint/src/main/java/com/fingerprintjs/android/fingerprint/info_providers/SensorsInfoProvider.kt [hunk 2]
```diff
@@ -18,18 +19,16 @@ public interface SensorDataSource {
...
     override fun sensors(): List<SensorData> {
-        return executeSafe(
-            {
-                sensorManager.getSensorList(Sensor.TYPE_ALL).map {
+        return safe {
+                sensorManager.getOrThrow().getSensorList(Sensor.TYPE_ALL)!!.filterNotNull().map {
                     SensorData(
-                        it.name,
-                        it.vendor
+                        it.name!!,
```
Comment (Alexey-Verkhovsky) [inline fingerprint/src/main/java/com/fingerprintjs/android/fingerprint/info_providers/SensorsInfoProvider.kt:28]: If one sensor will have null-name or vendor – then all the list becomes empty?

Comment (Sergey-Makarov) [reply fingerprint/src/main/java/com/fingerprintjs/android/fingerprint/info_providers/SensorsInfoProvider.kt]: Correct. This way the previous behaviour is preserved.

File: fingerprint/src/main/java/com/fingerprintjs/android/fingerprint/tools/logs/Messages.kt (hunk 1 above)
Comment (Sergey-Makarov) [reply fingerprint/src/main/java/com/fingerprintjs/android/fingerprint/tools/logs/Messages.kt]: Nice catch, fixed.

**Review by Alexey-Verkhovsky — APPROVED** [review]

**Pull request merged**